app/controllers/api/avatars.php

```diff
@@ -164,13 +169,20 @@
     ->label('scope', 'avatars.read')
     ->label('cache', true)
     ->label('cache.resource', 'avatar/credit-card')
-    ->label('sdk.auth', [APP_AUTH_TYPE_SESSION, APP_AUTH_TYPE_KEY, APP_AUTH_TYPE_JWT])
-    ->label('sdk.namespace', 'avatars')
-    ->label('sdk.method', 'getCreditCard')
-    ->label('sdk.methodType', 'location')
-    ->label('sdk.description', '/docs/references/avatars/get-credit-card.md')
-    ->label('sdk.response.code', Response::STATUS_CODE_OK)
-    ->label('sdk.response.type', Response::CONTENT_TYPE_IMAGE_PNG)
+    ->label('sdk', new Method(
+        namespace: 'avatars',
+        name: 'getCreditCard',
+        description: '/docs/references/avatars/get-credit-card.md',
+        auth: [AuthType::SESSION, AuthType::KEY, AuthType::JWT],
+        methodType: MethodType::LOCATION,
+        responses: [
+            new SDKResponse(
+                code: Response::STATUS_CODE_OK,
+                model: Response::MODEL_NONE,
+            )
+        ],
+        responseType: ResponseType::IMAGE_PNG
```

Comment:
If responses can be an array, the response type should probably be a property of `SDKResponse` so that each can define it's own type if needed